tests/queries/0_stateless/02406_minmax_behaviour.sql

```diff
@@ -0,0 +1,138 @@
+WITH
+    arrayJoin([1, 2, 3, nan, 4, 5]) AS data,
+    arrayJoin([nan, 1, 2, 3, 4]) AS data2,
+    arrayJoin([1, 2, 3, 4, nan]) AS data3,
+    arrayJoin([nan, nan, nan]) AS data4,
+    arrayJoin([nan, 1, 2, 3, nan]) AS data5
+SELECT
+    'minNan',
+    min(data),
+    min(data2),
+    min(data3),
+    min(data4),
+    min(data5);
+
+WITH
+    arrayJoin([1, 2, 3, nan, 4, 5]) AS data,
+    arrayJoin([nan, 1, 2, 3, 4]) AS data2,
+    arrayJoin([1, 2, 3, 4, nan]) AS data3,
+    arrayJoin([nan, nan, nan]) AS data4,
+    arrayJoin([nan, 1, 2, 3, nan]) AS data5
+SELECT
+    'maxNan',
+    max(data),
+    max(data2),
+    max(data3),
+    max(data4),
+    max(data5);
+
+SELECT 'minIf', minIf(number, rand() % 2 == 3) from numbers(10);
```

Comment:
Better to enable `echoOn` for these queries, to appear in `.reference` file and remove first projection column that suppose to identify query. It will be easier to check that everything is ok.

Reply:
:+1: